### PR TITLE
Update Readme with live link

### PR DESCRIPTION
**Pull request by awaisanwar544**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 > This is Portfolio project to showcase my work as a Developer.
 
 ## Live Link
-> https://awaisanwar544.github.io/portfolio/app/index.html
+[Click here to go to the live portfolio](https://awaisanwar544.github.io/portfolio/app/index.html)
```
Comment (Lordkaito) [inline README.md]: - [x] Please make sure that your link is working. Right now, the link it's showing a "404" error message, so please apply the necessary changes.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @awaisanwar544, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (awaisanwar544) [reply README.md]: updated to working link

**Review by Lordkaito — APPROVED** [review]
Hi @awaisanwar544, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
 
- [ ] [OPTIONAL] Please try to solve the merge conflicts.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**